.development.env

```diff
@@ -24,7 +24,6 @@ WEB_SERVER_THREADS=10
 WEB_SERVER_UWSGI_ASYNC_CORES=10
 CDN_URL=https://cdn.eq.gcp.onsdigital.uk
 CDN_ASSETS_PATH=/design-system
-ADDRESS_LOOKUP_API_URL=https://whitelodge-ai-api.census-gcp.onsdigital.uk
```

Comment:
I don't think we need to remove these as part of this ticket, it is not related to the census survey config. It is an env var for the lookups feature

Reply:
That makes sense, I've reverted it